src/provider/non_hiding_zeromorph.rs

```diff
@@ -443,19 +443,19 @@ fn eval_and_quotient_scalars<F: Field>(y: F, x: F, z: F, point: &[F]) -> (F, (Ve
   //           = [- (y^i * x^(2^num_vars - d_i - 1) + z * (x^(2^i) * Φ_(n-i-1)(x^(2^(i+1))) - u_i * Φ_(n-i)(x^(2^i)))), i ∈ [0, num_vars-1]]
   #[allow(clippy::disallowed_methods)]
   let q_scalars = iter::successors(Some(F::ONE), |acc| Some(*acc * y)).take(num_vars)
-    .zip_eq(offsets_of_x)
-    // length: num_vars + 1
-    .zip(squares_of_x)
-    // length: num_vars + 1
-    .zip(&vs)
-    .zip_eq(&vs[1..])
-    .zip_eq(point.iter().rev()) // assume variables come in BE form
-    .map(
-      |(((((power_of_y, offset_of_x), square_of_x), v_i), v_j), u_i)| {
-        (-(power_of_y * offset_of_x), -(z * (square_of_x * v_j - *u_i * v_i)))
-      },
-    )
-    .unzip();
+      .zip_eq(offsets_of_x)
```

Comment:
Weird that `cargo fmt` didn't complain with the original indentation!?

Reply:
Yes pretty strange, I kept the update as it seemed ok